### PR TITLE
BIG-254 Allow paritioned table in external bucket

### PR DESCRIPTION
**Pull request by jirkasemmler**
Jira: BIG-254
Connection PR: https://github.com/keboola/connection/pull/5037
SAPI PR: -

---

zatim bez connection, to pustim vecer. tldr - try catch na ten partitioning a kdyz tak to pustit

### CODE REVIEW

File: tests/data/hive/part=1/a_b_c-3row.csv [hunk 1]
```diff
@@ -0,0 +1,4 @@
+a,b,c
```
Comment (jirkasemmler) [inline tests/data/hive/part=1/a_b_c-3row.csv:1]: na hive test

File: tests/functional/UseCase/Info/ObjectInfoTest.php [hunk 2]
```diff
@@ -140,7 +140,7 @@ public function testInfoSchema(): void
         $objects = $response->getSchemaInfo()->getObjects()->getIterator();
 
         // can select from both external tables, because main project has access to files in GCS
-        $this->assertCount(8, $objects);
+        $this->assertCount(9, $objects);
```
Comment (jirkasemmler) [inline tests/functional/UseCase/Info/ObjectInfoTest.php:143]: vsude pribyl novy objekt - ta partitionovana tabulka

File: tests/functional/UseCase/Table/Import/FromFile/ImportTableFromFileTest.php [hunk 3]
```diff
@@ -393,6 +393,8 @@ public function testImportTableFromFileWithNullValues(): void
         );
         $dedupCols = new RepeatedField(GPBType::STRING);
         $convertCols = new RepeatedField(GPBType::STRING);
+        $importAsNull = new RepeatedField(GPBType::STRING);
```
Comment (jirkasemmler) [inline tests/functional/UseCase/Table/Import/FromFile/ImportTableFromFileTest.php:396]: fix spojeny s https://github.com/keboola/storage-backend/pull/150

**Comment by romanbracinik** [comment]
👁️

**Review by romanbracinik — APPROVED** [review]
👌

**Pull request merged**